### PR TITLE
ICMSLST-2673 Fix section 5 goods having hardcoded subsection in pdf.

### PR DESCRIPTION
**Pull request by MattHolmes123**
Added [another bug](https://uktrade.atlassian.net/browse/ICMSLST-2708) to revisit this as the same problem is in one of the reports.

Once I have written tests and fixed ICMSLST-2708 I hope to remove some of the duplication relating to fetching FA-SIL goods.

I believe there are at least 3 places that fetch FA-SIL goods places that could do with a refactor.

### CODE REVIEW

File: web/utils/pdf/utils.py [hunk 1]
```diff
@@ -223,6 +224,50 @@ def _get_fa_dfl_goods(application: DFLApplication) -> list[str]:
     ]
 
 
+# TODO: Revisit in ICMSLST-2708
+def _get_fa_sil_goods(application: SILApplication) -> Iterable[tuple[str, int | str]]:
+    """Return all related FA SIL goods"""
+
+    for g in application.goods_section1.filter(is_active=True):
+        section = "1"
```
Comment (mikeylockwood) [inline web/utils/pdf/utils.py:232]: Why set sections as variables instead of just hardcoding into the text if they don't change?

**Review by mikeylockwood — APPROVED** [review]

File: web/tests/utils/pdf/test_utils.py [hunk 2]
```diff
@@ -386,3 +387,64 @@ def test__get_importer_eori_numbers_wood(wood_app_submitted):
...
     assert _get_importer_eori_numbers(wood_app_submitted) == ["GB0123456789ABCDE"]
+
+
+def test__get_fa_sil_goods(fa_sil_app_submitted, ilb_admin_user):
+    expected = [
+        ("Section 1 goods to which Section 1 of the Firearms Act 1968, as amended, applies.", 111),
+        ("Section 2 goods to which Section 2 of the Firearms Act 1968, as amended, applies.", 222),
+        (
+            "Section 5 goods to which Section 5(A) of the Firearms Act 1968, as amended, applies.",
+            333,
```
Comment (mikeylockwood) [inline web/tests/utils/pdf/test_utils.py:398]: One of these tests should be for an unlimited quantity

**Review by mikeylockwood — APPROVED** [review]

File: web/utils/pdf/utils.py (hunk 1 above)
Comment (MattHolmes123) [reply web/utils/pdf/utils.py:232]: I plan on refactoring it in that other bug I raised.

File: web/tests/utils/pdf/test_utils.py (hunk 2 above)
Comment (MattHolmes123) [reply web/tests/utils/pdf/test_utils.py:398]: Will do that in the next ticket.

**Pull request merged**